From e11e30779d4c14b61c7726280494528fd26ae981 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Tue, 13 Mar 2012 12:50:28 +0530
Subject: [PATCH 172/182] ARM: OMAP2+: am33xx: Enable PWM support in defconfig

Enable the following support in defconfig
1. ECAP and EHRPWM based PWM support
2. PWM based backlight support.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/configs/am335x_evm_defconfig |    7 +++++--
 1 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/am335x_evm_defconfig b/arch/arm/configs/am335x_evm_defconfig
index e29bf78..43ac622 100644
--- a/arch/arm/configs/am335x_evm_defconfig
+++ b/arch/arm/configs/am335x_evm_defconfig
@@ -3,6 +3,7 @@
 # Linux/arm 3.2.0 Kernel Configuration
 #
 CONFIG_ARM=y
+CONFIG_HAVE_PWM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
 CONFIG_HAVE_SCHED_CLOCK=y
 CONFIG_GENERIC_GPIO=y
@@ -1222,6 +1223,7 @@ CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_TWL6040_VIBRA is not set
 # CONFIG_INPUT_UINPUT is not set
 # CONFIG_INPUT_PCF8574 is not set
+# CONFIG_INPUT_PWM_BEEPER is not set
 # CONFIG_INPUT_GPIO_ROTARY_ENCODER is not set
 # CONFIG_INPUT_ADXL34X is not set
 # CONFIG_INPUT_CMA3000 is not set
@@ -1396,8 +1398,8 @@ CONFIG_GPIO_SYSFS=y
 #
 CONFIG_GPIO_TPS65910=y
 CONFIG_GENERIC_PWM=y
-# CONFIG_DAVINCI_EHRPWM is not set
-# CONFIG_ECAP_PWM is not set
+CONFIG_DAVINCI_EHRPWM=y
+CONFIG_ECAP_PWM=y
 # CONFIG_W1 is not set
 # CONFIG_POWER_SUPPLY is not set
 CONFIG_HWMON=y
@@ -1792,6 +1794,7 @@ CONFIG_LCD_CLASS_DEVICE=y
 # CONFIG_LCD_AMS369FG06 is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 # CONFIG_BACKLIGHT_GENERIC is not set
+CONFIG_BACKLIGHT_PWM=y
 # CONFIG_BACKLIGHT_ADP8860 is not set
 # CONFIG_BACKLIGHT_ADP8870 is not set
 CONFIG_BACKLIGHT_TLC59108=y
-- 
1.7.9.1

